excellent/tools/context_walk.go

```diff
@@ -0,0 +1,37 @@
+package tools
+
+import (
+	"github.com/nyaruka/goflow/excellent/types"
+)
+
+// ContextWalk traverses the given context invoking the callback for each value
+func ContextWalk(context *types.XObject, callback func(types.XValue)) {
+	contextWalk(context, callback)
+}
+
+// ContextFindObjects traverses the given context invoking the callback for each object found
+func ContextFindObjects(context *types.XObject, callback func(*types.XObject)) {
```

Comment:
minor but the different between `walk` and `find` here feels a bit arbitrary.. when I saw find I was thinking, "oh, you pass in the type you want to get callbacks for".. to me `ContextWalkObjects` feels a bit more consistent / intuitive.

Reply:
Fine with that